static/js/components/NewDropDownSearchBar.jsx

```diff
@@ -0,0 +1,54 @@
+// NewDropDownSearchBar.jsx
```

Comment:
Maybe a more generic name here? It's basically a `SearchableDropDown`, not necessarily so select anything `New` as this name suggests.

Reply:
Thank you for the feedback!